cfme/tests/storage/test_volume.py

```diff
@@ -224,7 +227,30 @@ def test_storage_volume_attached_delete():
             4. check for flash message " Cloud Volume "Volume_name" cannot be
             removed because it is attached to one or more Instances "
         """
-    pass
+    volume = create_volume(appliance, provider, from_manager, az=instance_fixture.
+                           vm_default_args["environment"]["availability_zone"], should_assert=True)
+
+    # attach
+    volume.attach_instance(name=instance_fixture.name, mountpoint='/dev/sdm',
+                           from_manager=from_manager)
+    wait_for(lambda: volume.instance_count == 1, delay=15, timeout=600)
+
+    try:
+        volume.delete(from_manager=from_manager)
+        pytest.fail("Attached volume was deleted!")
+    except Exception:
+        if from_manager:
+            view = volume.browser.create_view(StorageManagerVolumeAllView,
```

Comment:
@mmojzis Is this view registered against an object?

Why do you need to create the view with the `browser`?

Reply:
This is because volume.create_view doesn't have additional_context keyword argument and I need to use this one to override object.

Reply:
@mmojzis Unless this is specific to this test and will not be used in the future, I think this view should be registered against the `volume` object, with the `from_manager` behavior taken care of by the `VIEW` property.

Reply:
Well I would have to rewrite all the other tests using this.
I have all volume/snapshot/backup tests parametrized as from_volume/from_manager and all these tests are using same functionality.

Reply:
To keep PR scope intact, this can be tackled in a separate PR